templates/posts.html

```diff
@@ -21,6 +21,25 @@ <h5 class="post-title">
           {{'updated '+ "{:%Y-%m-%d %H:%M}".format(post['post'].updated_on) or 'created ' +"{:%Y-%m-%d %H:%M}".format(post['post'].created_on)}}</small>
         <hr />
         <div>
+<<<<<<< HEAD
```

Comment:
this was a "merge conflict", you should double check because you're going to have some weir output